src/libmain.c

```diff
@@ -119,17 +119,17 @@ static gboolean dummy = FALSE;
 /* in alphabetical order of short options */
 static GOptionEntry entries[] =
 {
-	{ "column", 0, 0, G_OPTION_ARG_INT, &cl_options.goto_column, N_("Set initial column number for the first opened file (useful in conjunction with --line)"), NULL },
-	{ "config", 'c', 0, G_OPTION_ARG_FILENAME, &alternate_config, N_("Use an alternate configuration directory"), NULL },
+	{ "column", 0, 0, G_OPTION_ARG_INT, &cl_options.goto_column, N_("Set initial column number for the first opened file (useful in conjunction with --line)"), N_("COLUMN") },
+	{ "config", 'c', 0, G_OPTION_ARG_FILENAME, &alternate_config, N_("Use an alternate configuration directory"), N_("DIR") },
```

Comment:
I would then try and include the placeholder in the description, like *Use alternate configuration directory DIR* or alike.

Reply:
Done.